hydrus/hydraspec/doc_maker.py

```diff
@@ -16,56 +41,46 @@ def createDoc(doc: Dict[str, Any], HYDRUS_SERVER_URL: str=None, API_NAME: str=No
         raise SyntaxError("The API Documentation must have [@id]")
 
     # Extract base_url, entrypoint and API name
-    matchObj = re.match(r'(.*)://(.*)/(.*)/vocab#?', id_, re.M | re.I)
-    if matchObj:
-        base_url = matchObj.group(1) + '://' + matchObj.group(2) + '/'
-        entrypoint = matchObj.group(3)
+    match_obj = re.match(r'(.*)://(.*)/(.*)/vocab#?', id_, re.M | re.I)
+    if match_obj:
+        base_url = match_obj.group(1) + '://' + match_obj.group(2) + '/'
```

Comment:
String concatenation with `+` should be a rare occurrence. Wherever possible use string formatting `str.format` (rarely concatenation is preferred for performance reason).

Reply:
@Mec-iS Its updated as per your suggestion.